### DISCUSSION TITLE
Syntax for secrets.yaml for multiple cameras?

### DISCUSSION THREAD

**Discussion by parkroyk** [roflcoopter/viseron#586] [Q&A] [answered]
The documentation [here](https://viseron.netlify.app/docs/documentation/configuration#camera-domain) only lists an example for single camera setup. How do I indicate different secrets for different cameras? My search in Q&A surprisingly showed nothing? Thanks.

**Comment by parkroyk** [comment, marked as answer]
Ah I figured it out a minute after I posted :)

Basically the secrets.yaml is a map, so you can list whatever you want in there then reference it from config.yaml. e.g.

in secrets.yaml

camera_1_ip: 192.168.xxx.xx1
camera_1_username: username1
camera_1_password: password1
camera_2_ip: 192.168.xxx.xx2
camera_2_username: username2
camera_2_password: password2
.


and in config.yaml

camera:
  camera_1:
    name: bath room 1
    host: !secret camera_1_ip
    username: !secret camera_1_username
    password: !secret camera_1_username
    .
  camera_2:
    name: bath room 2
    host: !secret camera_2_ip
    username: !secret camera_2_username
    password: !secret camera_2_username
    .

and that works! Please update the doc to reflect - thanks.

**Reply by roflcoopter** [reply]
Great that you figured it out. Will fix the docs 👍
